src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommand.cs

```diff
@@ -154,21 +154,7 @@ private static ReportType GetReportType(bool isDeprecated, bool isOutdated, bool
                 return isDeprecated ? ReportType.Deprecated : isOutdated ? ReportType.Outdated : ReportType.Vulnerable;
             }
 
-            // We have a conflict - throw with appropriate message
-            var firstOption = string.Empty;
-            var incompatibleOption = string.Empty;
-            if (isOutdated)
-            {
-                firstOption = "--outdated";
-                incompatibleOption = isDeprecated ? "--deprecated" : "--vulnerable";
-            }
-            else if (isDeprecated)
-            {
-                firstOption = "--deprecated";
-                incompatibleOption = "--vulnerable";
-            }
-
-            throw new ArgumentException(string.Format(Strings.ListPkg_InvalidOptions, firstOption, incompatibleOption));
+            throw new ArgumentException(string.Format(Strings.ListPkg_InvalidOptions));
```

Comment:
Product changes and CI changes in the same PR can increase the effort required to cherry-pick onto old release branches. Consider splitting the PR. This change, for example, will cause product issues if cherry-picked onto a branch before `--deprecated` and `--vulnerable` were added.

Reply:
This was a test fix.
Honestly, this product change is a dead codepath. 

The parsing is happening on the SDK side, ours is there for our internal testing. In production scenarios, only the dotnet.exe code gets hit. 

Alternatively, I can just change the string that the test asserts from the resource to a hardcoded string.

lmk wyt.